.github/workflows/release-brew.yml

```diff
@@ -14,16 +14,15 @@ jobs:
     steps:
     - name: Install dafny
       run: |
-        brew install dafny || echo DONE
-        brew link --overwrite dotnet
```

Comment:
Was the intention here to overwrite a dotnet Dafny installation? Seems surprising since this actions job doesn't install Dafny through dotnet.

Reply:
It does seem surprising, which is why I removed it.